config/routes.rb

```diff
@@ -165,6 +165,7 @@ def call(env)
 
   namespace :provider, path: 'p', constraints: ProviderDomainConstraint do
     resource :password, :only => %i(new show update destroy)
+    get '/request_password_reset', to: 'passwords#request_password_reset'
```

Comment:
Following https://github.com/3scale/porta/pull/1765/files#r391747885, it would be:
```ruby
resource :password, only: %i[new show update destroy] do
  get 'reset'
 end
```